vault-rcp/src/main/java/org/apache/jackrabbit/vault/rcp/impl/RcpTaskImpl.java

```diff
@@ -211,18 +210,22 @@ public STATE getState() {
         return state;
     }
 
+    @Override
     public RepositoryAddress getSource() {
         return src;
     }
 
+    @Override
     public String getDestination() {
         return dst;
     }
 
+    @Override
     public void setRecursive(boolean b) {
         this.recursive = b;
```

Comment:
don't allow modification if the task is running.

Reply:
I will make recursive final